### DISCUSSION TITLE
Program with RtWlanE kext

### DISCUSSION THREAD

**Discussion by slowbeatsua** [chris1111/Wireless-USB-Big-Sur-Adapter#309] [Ideas]
Hello, chris1111. Please do a program with RtWlanE kext, which Realtek posted recently because some people can`t use their PCIe Realtek adapters in macOS and Hackintosh. Thanks for understanding!

**Comment by chris1111** [comment]
I  don’t think a PCIe WiFi is compatible new macOS systems

**Reply by slowbeatsua** [reply]
I checked kext, installed it in LE, and it works

**Reply by slowbeatsua** [reply]
Via your WirelessBigSurAdapter

**Reply by slowbeatsua** [reply]
Then can you create the program?

**Comment by chris1111** [comment]
WirelessBigSurAdapter is already created
